ldap/servers/plugins/replication/windows_protocol_util.c

```diff
@@ -4381,8 +4371,9 @@ windows_create_local_entry(Private_Repl_Protocol *prp, Slapi_Entry *remote_entry
         /* Fatal error : need the guid */
         goto error;
     }
-    /* Hack for NT4, which has no surname */
-    if (is_nt4 && is_user) {
+
```

Comment:
Just think of all those sad nt4 dcs from 1996 you are just throwing out support for 🥺

Reply:
> Just think of all those sad nt4 dcs from 1996 you are just throwing out support for pleading_face

I know, it's a sad day indeed ;-)